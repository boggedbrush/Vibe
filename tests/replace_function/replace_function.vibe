# VibeSpec: 1.6
patch_type: replace_function
file: hello.py
name: farewell
--- code: |
    def farewell(name):
        print(f"So long, and thanks for all the fish, {name}!")