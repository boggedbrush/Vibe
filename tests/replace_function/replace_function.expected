# hello.py
def greet(name):
    print(f"Hello, {name}!")


def farewell(name):
    print(f"So long, and thanks for all the fish, {name}!")
